elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/framework/SQLUnitTest.java

```diff
@@ -385,11 +385,11 @@ public static void init(String sqlDialect) {
         playerStatsTable = (SQLTable) metaDataStore.getTable("playerStats", NO_VERSION);
     }
 
-    private static String getCompatabilityMode(String dialect) {
-        if (dialect.equals(SQLDialectFactory.getMySQLDialect().getDialectType())) {
-            return "MODE=MySQL;DATABASE_TO_LOWER=TRUE";
-        } else if (dialect.equals(SQLDialectFactory.getPostgresDialect().getDialectType())) {
-            return "MODE=PostgreSQL;DATABASE_TO_LOWER=TRUE";
+    private static String getCompatabilityMode(String dialectType) {
+        if (dialectType.equals(SQLDialectFactory.getMySQLDialect().getDialectType())) {
```

Comment:
Hikari doesn't support `DATABASE_TO_LOWER` property. Earlier `if` block was not getting executed as we were trying to compare class name with dialect type.